handler/handler_test.go

```diff
@@ -551,3 +560,40 @@ func TestSplitLabels(t *testing.T) {
 		})
 	}
 }
+
+func TestWipeMetricStore(t *testing.T) {
+	// Create MockMetricStore with a few GroupingKeyToMetricGroup metrics
+	// so they can be returned by GetMetricFamiliesMap() to later send write
+	// requests for each of them.
+	metricCount := 5
+	mms := newMockMetricStore()
+	for i := 0; i < metricCount; i++ {
+		mms.metricGroups[uint64(i)] = storage.MetricGroup{}
+	}
```

Comment:
Adding to my comment above, I'd write this  in the following way to avoid the need for a constructor:

```Go
	metricCount := 5
	mgs := storage.GroupingKeyToMetricGroup{}
	for i := 0; i < metricCount; i++ {
		mgs[uint64(i)] = storage.MetricGroup{}
	}
	mms := MockMetricStore{metricGroups: mgs}
```

In total, that's fewer lines (no constructor) and makes things more explicit.

Reply:
:+1: